lib/test/clojure_lsp/refactor/transform_test.clj

```diff
@@ -257,7 +270,14 @@
   (testing "within coll"
     (is (= (h/code "(do (let [f 1]"
                    "     [f]))")
-           (expand-let "(do [(let [f 1] |f)])")))))
+           (expand-let "(do [(let [f 1] |f)])"))))
+  (testing "multi-arity fn"
+    (testing "expanding to top"
+      (is (= (h/code "(defn foo (let [a 1]"
```

Comment:
We could not allow that but it's ok since we are actually expanding and may make sense if a function is inside another block of code for some reason, probably not a big deal right now